ksql-parser/src/main/java/io/confluent/ksql/parser/tree/ExecutableDdlStatement.java

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright 2017 Confluent Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ **/
+
+package io.confluent.ksql.parser.tree;
+
+/**
+ * A Marker interface to signal that a Statement is an executable DDLStatement
+ */
+public interface ExecutableDdlStatement extends DdlStatement {
```

Comment:
Added to differentiate between DDL statements the Engine can execute and those it can't.